graphs/generate_graphs.py

```
@@ -5,201 +5,286 @@
-
-# Initialize translations globally
-translations = load_translations(config['LANGUAGE'])
-
-# Define consistent colors
-TV_COLOR = config['TV_COLOR'].strip('"')
-MOVIE_COLOR = config['MOVIE_COLOR'].strip('"')
-ANNOTATION_COLOR = config['ANNOTATION_COLOR'].strip('"')
 
 # Helper function to fetch data from Tautulli
-def fetch_tautulli_data(cmd, params={}):
+def fetch_tautulli_data(cmd, params={}, config=None):
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)